python/openai/tests/test_chat_completions.py

```diff
@@ -199,14 +201,21 @@ def test_chat_completions_invalid_sampling_parameters(
         assert response.status_code == 422
 
     # Simple tests to verify max_tokens roughly behaves as expected
+    @pytest.mark.parametrize(
+        "param_key",
```

Comment:
Let's call this something a little more intuitive to read, like `max_tokens_key` or something.

Reading this isn't clear at a glance what kind of param it is for:
```python
        payload[param_key] = 1
```